src/pyOpenMS/pxds/FeatureFinderAlgorithmMetaboIdent.pxd

```diff
@@ -74,6 +74,63 @@ cdef extern from "<OpenMS/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderAlgorithmMe
             libcpp_vector[ double ] rts,
             libcpp_vector[ double ] rt_ranges,
             libcpp_vector[ double ] iso_distrib) nogil except +
-        
         # wrap-doc:
-        #    Represents a compound in the ID table
+        #     Represents a compound in the in the FeatureFinderMetaboIdent library table.
```

Comment:
Just as a note, after listing the members of the class, autowrap automatically creates __getX__ methods in python and therefore obj.member should work.
But this is fine, too.